### PR TITLE
feat(St Tree): St Tree styles

### PR DESCRIPTION
**Pull request by acuevas-stratio**
St Tree styles

## TYPE
- Bugfix, New component, New service, Utils, etc.

## Description
Add a description of your changes here.

### Documentation
- [ ] Have explained API (Inputs, Outpus, Models, Functions, etc)
- [ ] Have example running
- [ ] Have example code (HTML, ts)
- [ ] Have changelog.md updated

### Code
- [x] Have 3 spaces indentation
- [x] Pass ts lint Task
- [x] Pass Sass lint Task
- [ ] Have QA-tags

### Tests
- [x] Have 70% Tests coverage

### Other observations
- Add other information you consider important

### Post-review reminder
- Remember reviewer approve changes inside the Files Changed tab (Review changes dropdown)
- Squash commits
- Release version in case of breaking changes and notify of all front

### CODE REVIEW

File: src/lib/st-tree/demo/st-tree-demo.ts [hunk 1]
```diff
@@ -78,6 +79,8 @@ export class StTreeDemoComponent {
       this.notificationChangeStream = this.subject.asObservable();
    }
 
+   private selectedPath: string;
```
Comment (ppenalver) [inline src/lib/st-tree/demo/st-tree-demo.ts]: Must be before constructor

File: src/lib/st-tree/demo/st-tree-demo.ts [hunk 2]
```diff
@@ -86,7 +89,16 @@ export class StTreeDemoComponent {
    }
 
    onSelectNode(nodeChange: StNodeTreeChange): void {
-      console.log('select node', nodeChange);
+      console.log('selected node', nodeChange);
+      let node: StNodeTree;
+      if (this.selectedPath) {
+         node = _get<StNodeTree>(this.tree, this.selectedPath, this.tree);
+         node.selected = false;
+      };
```
Comment (ppenalver) [inline src/lib/st-tree/demo/st-tree-demo.ts]: unnecesary semicolon

File: src/lib/st-tree/st-node-tree/st-node-tree.component.spec.ts [hunk 3]
```diff
@@ -262,6 +262,24 @@ describe('StTreeComponent', () => {
...
+         comp.node.selected = true;
+         fixture.detectChanges();
+         expect(comp.isNodeSelected()).toBeTruthy();
+      });
+
+      it('Selected path is equal to received path', () => {
+         comp.father = [0, 1];
+         comp.pos = 4;
+         comp.node = mockTree;
+         comp.selectedPath = "children[1].children[4]";
```
Comment (ppenalver) [inline src/lib/st-tree/st-node-tree/st-node-tree.component.spec.ts]: " should be '

File: src/lib/st-tree/demo/st-tree-demo.ts (hunk 1 above)
Comment (acuevas-stratio) [inline src/lib/st-tree/demo/st-tree-demo.ts]: There is no constructor in this class

**Pull request merged**